pom.xml

```diff
@@ -115,6 +115,41 @@
             </includes>
           </configuration>
         </plugin>
+        
+        <plugin>
+          <artifactId>maven-checkstyle-plugin</artifactId>
+          <version>2.17</version>
+          <dependencies>
+            <dependency>
+              <groupId>com.spotify.checkstyle</groupId>
+              <artifactId>spotify-checkstyle-config</artifactId>
+              <version>1.0.2</version>
+            </dependency>
+            <dependency>
+              <groupId>com.puppycrawl.tools</groupId>
+              <artifactId>checkstyle</artifactId>
+              <version>6.19</version>
+            </dependency>
+          </dependencies>
+          <configuration>
+            <configLocation>spotify_checks.xml</configLocation>
```

Comment:
do you really need to point out the file? it's not even in this repo so it's kind of confusing.